Extend docs with Page::retrieveExtendedClientDetails method info (#407)
vaadin/docs: articles/flow/advanced/browser-access.asciidoc

});
----

// TODO add documentation on VaadinSession.getCurrent().getBrowser()

> == Getting the extended Client Side Details

Reviewer: ```suggestion
== Getting the Extended Client-Side Details
```
Looks like Vale crashed and didn't complain a lot of these.

> Use the [methodname]`Page.retrieveExtendedClientDetails` method to retrieve extended client side details.

Reviewer: ```suggestion

Use [methodname]`retrieveExtendedClientDetails()` in the current [classname]#Page# object to retrieve extended client-side details.
```

> The method accepts a callback which being called on a client response provides an [classname]#ExtendedClientDetails# 

Reviewer: ```suggestion
The method accepts a callback that, when called after client response, gets an [classname]#ExtendedClientDetails# object with various client-side data.
```

> object with various client side data. E.g. this code snippet can be used to get the screen width:

Reviewer: ```suggestion

For example, the following can be used to get the screen width:
```
